website/docs/r/compute_cluster_vm_dependency_rule.html.markdown

```diff
@@ -0,0 +1,176 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_compute_cluster_vm_dependency_rule"
+sidebar_current: "docs-vsphere-resource-compute-cluster-vm-dependency-rule"
+description: |-
+  Provides a VMware vSphere cluster VM dependency rule. This can be used to manage VM dependency rules for vSphere HA.
+---
+
+# vsphere\_compute\_cluster\_vm\_dependency\_rule
+
+The `vsphere_compute_cluster_vm_dependency_rule` resource can be used to manage
+VM dependency rules in a cluster, either created by the
+[`vsphere_compute_cluster`][tf-vsphere-cluster-resource] resource or looked up
+by the [`vsphere_compute_cluster`][tf-vsphere-cluster-data-source] data source.
+
+[tf-vsphere-cluster-resource]: /docs/providers/vsphere/r/compute_cluster.html
+[tf-vsphere-cluster-data-source]: /docs/providers/vsphere/d/compute_cluster.html
+
+A virtual machine dependency rule applies to vSphere HA, and allows
+user-defined startup orders for virtual machines in the case of host failure.
+Virtual machines and hosts are supplied via groups, which can be managed via
```

Comment:
I think this may be a copy/paste oversight from vm host rules. I don't think hosts are needed for this rule.